src/modules/tests/test-status/test-status.actions.ts

```diff
@@ -1,30 +1,39 @@
 import { Action } from '@ngrx/store';
 
-// TODO: These actions are not relatable to an action in the app. Rename needed.
+export const SET_TEST_STATUS_BOOKED = '[???] Set Test Status to Booked';
```

Comment:
Need a name for these - can't it just be [ Test Status ]

Reply:
done